#parameters (SRC2)
block_matcher:
  block_size: 11
  num_disparities: 112
  min_disparity: 0
  max_disparity: 6
  uniqueness_ratio: 32
  speckle_window_size: 150
  speckle_range: 2
  prefilter_cap: 13
  prefilter_size: 0
  use_semi_global: 1

good_features_to_track:
  max_features: 1000
  quality: 0.001
  min_distance: 5 #3
  block_size: 5 #3
  use_harris_detector: false
  k: 0.04

tracker:
  max_iter: 30
  eps: 0.01
  window: 20 #20
  max_level: 4 #3
  max_age: 10

frame_ids:
  camera_frame_id: "scout_1_tf/left_camera_optical"
  base_frame_id: "scout_1_tf/base_footprint"

vo:
  publish_tracks: true
  
#parameters (KITTI)
# block_matcher:
#   block_size: 15
#   num_disparities: 64
#   min_disparity: 0
#   max_disparity: 12
#   uniqueness_ratio: 32
#   speckle_window_size: 200
#   speckle_range: 2
#   prefilter_cap: 0
#   prefilter_size: 0
#   use_semi_global: 1

# good_features_to_track:
#   max_features: 2000
#   quality: 0.001
#   min_distance: 10
#   block_size: 5
#   use_harris_detector: false
#   k: 0.04

# tracker:
#   max_iter: 30
#   eps: 0.01
#   window: 20
#   max_level: 4
#   max_age: 10